tests/LogicItLab/Salesforce/MapperBundle/WsdlValidatorTestCase.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace LogicItLab\Salesforce\MapperBundle;
+
+use LogicItLab\Salesforce\MapperBundle\Annotation\AnnotationReader;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+class WsdlValidatorTestCase extends KernelTestCase
+{
+    /** @var KernelInterface */
+    protected $bootedKernel;
+
+    /** @var WsdlValidator */
+    private $wsdlValidator =  null;
+
+    public function testWsdlIsValid()
+    {
+        $this->assertInstanceOf(WsdlValidator::class, $this->wsdlValidator, "Validator should be booted before run a test.");
```

Comment:
"Validator should be booted before run a test." should be "Validator should be booted before a test run"

Reply:
I guess the setUp is missing